src/testUtils/dataBuilders/RowsBuilder.js

```diff
@@ -0,0 +1,28 @@
+import { RowBuilder } from './RowBuilder';
+
+export class RowsBuilder {
```

Comment:
So "RowsBuilder" imports "RowBuilder"?

I guess those builder are too complex and don't provide anything useful.
I would go with simple objects.

Even if we cannot have TS here.